src/gwt/src/org/rstudio/core/client/files/FileSystemItem.java

```diff
@@ -385,6 +385,7 @@ public final native boolean exists() /*-{
       MIME_TYPES.put( "rmd",   "text/x-r-markdown");
       MIME_TYPES.put( "rhtml", "text/x-r-html");
       MIME_TYPES.put( "rpres", "text/x-r-presentation");
+      MIME_TYPES.put( "nb.html", "text/x-r-notebook");
```

Comment:
Again I don't think it's a problem for .nb.html to have a MIME type of just plain HTML.